WebDriverAgentLib/Utilities/FBScreen.m

```diff
@@ -19,7 +19,7 @@ + (double)scale
   return [XCUIScreen.mainScreen scale];
 }
 
-+ (CGSize)statusBarSizeForApplication:(XCUIApplication *)application
++ (CGSize)statusBarSizeForApplication
```

Comment:
I think we can safely remove iOS 13 condition as we anyway don't support it for a long time